src/models.ts

```diff
@@ -320,12 +319,16 @@ export class Defaults {
 	}
 }
 
+class EnvironmentMap {
+	[key: string]: FeatureFlagConfig | FlagConfiguration;
```

Comment:
What's the difference between `FeatureFlagConfig` and `FlagConfiguration`? Is one of these the API result and the other the stream result? Perhaps we could discard the API results since we're always processing updates from the stream? Ideally we would minimize the amount of redundant types and data

Reply:
Yes that's it exactly. It might be possible to discard it but I'm not 100% certain at this time. With this refactor overall we are drastically reducing redundant data. We are merging the targeting with the API Flag Summary. So there is no duplication of data at that point.